website/src/pages/extensions/index.tsx

```
@@ -1,24 +1,682 @@
+                    target.style.display = 'none';
+                  }}
+                />
+              </div>
+              <div>
+                <h2 className="text-3xl font-bold text-charcoal-300 dark:text-white mb-2">{extension.name}</h2>
+                <p className="text-lg text-gray-600 dark:text-gray-300 -mt-1 mb-0">by {extension.publisher}</p>
+              </div>
+            </div>
+            <button
+              onClick={onClose}
+              className="text-gray-500 hover:text-gray-700 dark:text-gray-400 dark:hover:text-gray-200 cursor-pointer">
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)